### ISSUE TITLE
Questions about RNN example

### ISSUE THREAD

**Issue by yixin94** [aymericdamien/TensorFlow-Examples#288]
https://github.com/aymericdamien/TensorFlow-Examples/blob/9e1bb504f5d0f209d000997ce2ad95bb891798ab/examples/3_NeuralNetworks/recurrent_network.py#L64

I am confused about `lstm_cell`, which is a local variable in the function `RNN(x, weights, biases)`. Each time we call `logits = RNN(X, weights, biases)`, I feel that the `lstm_cell` becomes a new local variable created in the function and the weights in `lstm_cell` are always of initialized value. I wonder if the `lstm_cell` and `rnn.static_rnn(lstm_cell, x, dtype=tf.float32)` should be global variables outside of the function `RNN(x, weights, biases)`.
